helm/idol-library-example-test/templates/ingress.yaml

```diff
@@ -15,4 +15,8 @@
     "destination" "idolacitest.ingress")  -}}
 {{- define "idolacitest.ingress" -}}
 {{/* Any View specific merges to base template go here */}}
+{{- end }}
+{{ if .Values.ingress.TLS.enabled -}}
+---
+{{ include "idol-library.ingress.secret" (dict "tls" .Values.ingress.TLS) -}}
 {{- end -}}
```

Comment:
I _think_ it's  possible to move this logic within the existing `idol-library.ingress` template and then no chart specific modifications will be required.

Reply:
See https://github.com/opentext-idol/idol-containers-toolkit/pull/60/files#r1663781401